health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/components/GenericChart.js

```
@@ -0,0 +1,171 @@
+      document.addEventListener("mousedown", handleClickOutside);
+    } else {
+      document.removeEventListener("mousedown", handleClickOutside);
+    }
+
+    return () => {
+      document.removeEventListener("mousedown", handleClickOutside);
+    };
+  }, [dropdownStatus]);
+
+  function download(data) {
+    let chartIdToDownload = children?.props?.data?.id || children?.props?.chartId || null;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'chartIdToDownload' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)